docs/spec.md

```diff
@@ -1384,6 +1384,13 @@ in the `operand` tensor and produces a `result` tensor. More formally,
 * (C5) For all `d` in `axes(operand)`:
   * `dim(operand, d) = 1` or
   * `dim(operand, d) = dim(result, broadcast_dimensions[d])`.
+* (C6) `is_per_tensor_quantized(operand) = is_per_tensor_quantized(result)`.
+* (C7) If `is_per_axis_quantized(result)`:
+  * `quantized_dimension(result) = broadcast_dimensions[quantized_dimension(operand)]`.
+  * If `dim(operand, quantized_dimension(operand)) = 1`, then
+    `scales(result)[i] = scales(operand)[0] and zero_points(result)[i] =
+    zero_points(operand)[0] for i in
+    range(dim(result, quantized_dimension(result)))`.
```

Comment:
totally optional - can we have an example in the spec that demonstrates the case for per-axis quantized tensor? The constraints might not be very intuitive for first-time users.

Reply:
Hi @dansuh17 thanks for the suggestion. 
We do have examples for per-axis quantization for `uniform_quantize` and `uniform_dequantize` ([here](https://github.com/openxla/stablehlo/blob/main/docs/spec.md#uniform_quantize) and [here](https://github.com/openxla/stablehlo/blob/main/docs/spec.md#uniform_dequantize)). For most of the other ops which supports quantized types in addition to other regular element types, we follow the norm to provide example for only one regular element type (ref point 6 of [this](https://github.com/openxla/stablehlo/blob/main/docs/spec_checklist.md#stablehlo-specification-checklist)). While we add constraint checks to the verifier for each op, we will be adding example for each  constraint check. Does that sound like a plan to you?

Reply:
SGTM! Will the verifier implementation follow this commit?

Reply:
Yes, that is right.